src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -304,7 +367,14 @@ public Person generatePerson(int index, long personSeed) {
     } catch (Throwable e) {
       // lots of fhir things throw errors for some reason
       e.printStackTrace();
-      throw e;
+      try {
+        return generatePerson(index, personSeed);
```

Comment:
I don't think this was meant to be here - if you catch an exception and call the function again you'll hit the same exception and wind up in an infinite loop